src/sql/connection/connection.py

```diff
@@ -492,7 +492,7 @@ def _get_database_information(self):
         pass
 
     @abc.abstractmethod
-    def to_table(self, table_name, data_frame, if_exists, index):
+    def to_table(self, table_name, data_frame, if_exists, index, schema):
```

Comment:
let's set schema=None for consistency

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/947/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R495-R496)